emanifest-py/src/emanifest/client.py

```diff
@@ -767,10 +707,10 @@ def GetAttachmentsReg(self, mtn):
             attachments: PDF and HTML files containing additional manifest information (such as scans or electronic copies) for the given MTN
               message of success or failure
         """
-        attach = requests.get(self.base_url + '/api/v1/state/emanifest/manifest/' + mtn + '/attachments',
+        resp = requests.get(self.base_url + '/api/v1/state/emanifest/manifest/' + mtn + '/attachments',
                               headers={'Accept': 'multipart/mixed', 'Authorization': 'Bearer ' + self.token},
                               stream=True)
-        if attach.ok:
+        if resp.ok:
             multipart_data = decoder.MultipartDecoder.from_response(attach)
```

Comment:
"attach needs to be changed to "resp" I believe
```suggestion
            multipart_data = decoder.MultipartDecoder.from_response(resp)
```